package.json

```diff
@@ -11,7 +11,7 @@
     "lint:sass": "NODE_ENV=production sass-lint -v --max-warnings 0",
     "postinstall": "npm prune && npm run build",
     "starterapp": "webpack-dev-server --config webpack/starterApp.js --devtool eval --progress --colors --hot --history-api-fallback",
-    "test": "NODE_ENV=test karma start",
+    "test": "karma start",
```

Comment:
Does karma set `NODE_ENV` for us or something?  Why wouldn't we need to set it here?